staff/mgarciarojas/Javascript exercises/multiplicationTable.js

```diff
@@ -0,0 +1,15 @@
+/**Multiplications table
+Write a function that writes in the console the multiplication table (from 1 to 10) */
+
+function multiplicationTable (){
+    var num;
```

Comment:
Let's try from the beginning to be SUPER-semantic when naming variables. A better name for this variable would be `numCell` or `numCellMultiplicationTable`

Reply:
Done